tests/node/rpc/messages/GetVehicleDataResponseTests.js

```diff
@@ -32,12 +39,23 @@ describe('GetVehicleDataResponseTests', function () {
             [GearStatus.KEY_TRANSMISSION_TYPE]: TransmissionType.MANUAL,
         };
 
+        this.climateData = new ClimateData()
```

Comment:
You can delete lines 42-50 `(this.climateData, and JSON_CLIMATE_DATA)` because these would be defined in `Test.js`.

Reply:
Fixed in d1e6617